### DISCUSSION TITLE
Build failing in Export .ipa step

### DISCUSSION THREAD

**Discussion by codemagic-safenow** [codemagic-ci-cd/codemagic-docs#1559] [Q&A] [answered]
Hey everyone! 

Had two builds fail with the following error: 

```
Failed to archive ios/Runner.xcworkspace. The following build commands failed:

PhaseScriptExecution [CP]\ Embed\ Pods\ Frameworks /Users/builder/Library/Developer/Xcode/DerivedData/Runner-dmdaxhihklevipgcsvxydmaouizc/Build/Intermediates.noindex/ArchiveIntermediates/dev/IntermediateBuildFilesPath/Runner.build/Release-dev-iphoneos/Runner.build/Script-C8B4EDEC60A24F121513E98E.sh (in target 'Runner' from project 'Runner')
	...
	sending incremental file list
	rsync: [sender] change_dir "/Users/builder/clone/business_app/ios/../../../IntermediateBuildFilesPath/UninstalledProducts/iphoneos" failed: No such file or directory (2)
	sent 19 bytes  received 12 bytes  62.00 bytes/sec
	total size is 0  speedup is 0.00
	rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1356) [sender=3.2.7]
	Command PhaseScriptExecution failed with a nonzero exit code
```

Anyone has an idea to resolve this?

**Comment by himesh-cm** [comment]
@codemagic-safenow, can you please share a build id?

**Reply by codemagic-safenow** [reply]
https://codemagic.io/app/609298cc2149abd440e1730c/build/643522d26da92a41c8123011
https://codemagic.io/app/609298cc2149abd440e1730c/build/64351645577c48ca2ea9f9c2

**Reply by himesh-cm** [reply]
Hi @codemagic-safenow, 
Can you share the contents of the Run Script & the Embed Pods Frameworks part from your Xcode local environment?
Also, you are on a paid plan; you should see the intercom widget at the bottom right; it is better to connect us via intercom for faster response.

**Comment by danielSafeNow** [comment, marked as answer]
This was fixed for me with the help of @himesh-cm, thanks again! 

The two options to fix it: 
- Go back to Xcode 14.2 
- Switch to Flutter 3.7.10

Both worked for me 👍
